docs/content/reference/cli/glooctl_debug.md

```diff
@@ -5,7 +5,16 @@ weight: 5
 ---
 ## glooctl debug
 
-Debug a Gloo resource (requires Gloo running on Kubernetes)
+Debug Gloo Gateway (requires Gloo running on Kubernetes)
```

Comment:
do we need all these glooctl changes?

Reply:
Not really, backed those changes and pulled in the one method needed by the backported code